### PR TITLE
39 Added stats file for shortreads and longreads

### PR DESCRIPTION
**Pull request by vlilanl**
Updated [shortreadsqc ](https://github.com/microbiomedata/ReadsQC/blob/39-stats-files/shortReadsqc.wdl)and [longreadsqc](https://github.com/microbiomedata/ReadsQC/blob/39-stats-files/longReadsqc.wdl) to include the correct stats.json output needed for automation

**Jaws tests**
Short Reads: 90023
Long Reads: 90324

### CODE REVIEW

File: Docker/BBtools_Dockerfile.txt [hunk 1]
```diff
@@ -3,15 +3,15 @@ FROM ubuntu:20.04
 LABEL base.image="ubuntu:20.04"
 LABEL dockerfile.version="1"
 LABEL software="BBTools"
-LABEL software.version="39.01"
+LABEL software.version="39.10"
```
Comment (aclum) [inline Docker/BBtools_Dockerfile.txt:6]: jgi is on 39.03.

File: docs/index.rst [hunk 2]
```diff
@@ -98,79 +98,76 @@ Inputs
...
+    Ex: **Interleaved**: "rqcfilter.input_files":[“first-int.fastq”,”second-int.fastq”]; **Non-Interleaved**: "rqcfilter.input_fq1": [“first-int-R1.fastq”,”second-int-R1.fastq”], "rqcfilter.input_fq2": [“first-int-R2.fastq”,”second-int-R2.fastq”]
 
 
 Output
 ------
 
 A directory named with the prefix of the FASTQ input file will be created and multiple output files are generated; the main QC FASTQ output is named prefix.anqdpht.fastq.gz. Using the dataset above as an example, the main output would be named SRR7877884-int-0.1.anqdpht.fastq.gz. Other files include statistics on the quality of the data; what was trimmed, detected, and filtered in the data; a status log, and a shell script documenting the steps implemented so the workflow can be reproduced.
 
-An example output JSON file (filterStats.json) is shown below:
+An example output JSON file (filterStats.txt) is shown below:
```
Comment (aclum) [inline docs/index.rst:135]: replace JSON with text

File: docs/index.rst [hunk 3]
```diff
@@ -98,79 +98,76 @@ Inputs
...
+	qtrimmedBases=0
+	qfilteredReads=208
+	qfilteredBases=10798
+	ktrimmedReads=456
+	ktrimmedBases=7726
+	kfilteredReads=0
+	kfilteredBases=0
+	outputReads=249398
+	outputBases=37003919
+
```
Comment (aclum) [inline docs/index.rst:151]: delete extra empty line, i dont know why this in happening in the actual files.

File: Docker/BBtools_Dockerfile.txt (hunk 1 above)
Comment (chienchi) [reply Docker/BBtools_Dockerfile.txt]: updated

**Review by chienchi — APPROVED** [review]

**Comment by vlilanl** [comment]
Jaws tests with updates:
Shortreads: 90405
Longreads: 90406

**Comment by aclum** [comment]
Very minor but can you pipe the long reads stats into pprintjson before dumping to nmdc_xxxxxxx_stats.json so it is nicely formatted like the short reads?

**Comment by vlilanl** [comment]
Jaws test: 90492

**Review by aclum — APPROVED** [review]
Looks great!

**Pull request merged**